src/chocolatey/infrastructure.app/nuget/NugetCommon.cs

```diff
@@ -240,15 +240,15 @@ public static IEnumerable<SourceRepository> GetRemoteRepositories(ChocolateyConf
             return repositories;
         }
 
-        public static IReadOnlyList<NuGetEndpointResources> GetRepositoryResources(ChocolateyConfiguration configuration, ILogger nugetLogger, IFileSystem filesystem)
+        public static IReadOnlyList<NuGetEndpointResources> GetRepositoryResources(ChocolateyConfiguration configuration, ILogger nugetLogger, IFileSystem filesystem, ChocolateySourceCacheContext cacheContext)
         {
             IEnumerable<SourceRepository> remoteRepositories = GetRemoteRepositories(configuration, nugetLogger, filesystem);
-            return GetRepositoryResources(remoteRepositories);
+            return GetRepositoryResources(remoteRepositories, cacheContext);
         }
 
-        public static IReadOnlyList<NuGetEndpointResources> GetRepositoryResources(IEnumerable<SourceRepository> packageRepositories)
+        public static IReadOnlyList<NuGetEndpointResources> GetRepositoryResources(IEnumerable<SourceRepository> packageRepositories, ChocolateySourceCacheContext cacheContext)

```

Comment:
This was missed initially, we need to add this as an overload and make the old method obsolete.